kaldb/src/test/java/com/slack/kaldb/logstore/search/LogIndexSearcherImplTest.java

```diff
@@ -424,15 +580,38 @@ public void testDisabledFullTextSearch() {
     strictLogStoreWithoutFts.logStore.commit();
     strictLogStoreWithoutFts.logStore.refresh();
 
-    SearchResult<LogMessage> babyInAll =
-        strictLogStoreWithoutFts.logSearcher.search(
-            TEST_DATASET_NAME, "_all:baby", 0, MAX_TIME, 1000, 1);
-    assertThat(babyInAll.hits.size()).isEqualTo(0);
+    assertThat(
+            strictLogStoreWithoutFts
+                .logSearcher
+                .search(TEST_DATASET_NAME, "_all:baby", 0, MAX_TIME, 1000, 1)
+                .hits
+                .size())
+        .isEqualTo(0);
 
-    SearchResult<LogMessage> numberInAll =
-        strictLogStoreWithoutFts.logSearcher.search(
-            TEST_DATASET_NAME, "_all:1234", 0, MAX_TIME, 1000, 1);
-    assertThat(numberInAll.hits.size()).isEqualTo(0);
+    assertThat(
+            strictLogStoreWithoutFts
+                .logSearcher
+                .search(TEST_DATASET_NAME, "_all:1234", 0, MAX_TIME, 1000, 1)
+                .hits
+                .size())
+        .isEqualTo(0);
+
+    // Without the _all field as default.
+    assertThat(
+            strictLogStoreWithoutFts
+                .logSearcher
+                .search(TEST_DATASET_NAME, "baby", 0, MAX_TIME, 1000, 1)
+                .hits
+                .size())
+        .isEqualTo(0);
+
+    assertThat(
+            strictLogStoreWithoutFts
+                .logSearcher
+                .search(TEST_DATASET_NAME, "1234", 0, MAX_TIME, 1000, 1)
+                .hits
+                .size())
+        .isEqualTo(0);
```

Comment:
Probably would be good to include the test asserts in the full text search disabled case as well, and not just the enabled one.

Reply:
Most of the additional asserts are related to how query parser handles the queries and not the all field. So, those tests are not as relevant here. However, added them here for completeness. 

We also have simpler higher level test at the chunk level, so we are covered there.